framework/include/base/FEProblem.h

```diff
@@ -406,7 +406,6 @@ class FEProblem :
 
   // Function /////
   virtual void addFunction(std::string type, const std::string & name, InputParameters parameters, bool auto_parsed = false);
-  virtual bool hasFunction(const std::string & name, THREAD_ID tid = 0);
```

Comment:
I will put it back.

On Tue, Jan 19, 2016 at 2:22 PM, David Andrs <notifications@github.com>
wrote:

> In framework/include/base/FEProblem.h
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_idaholab_moose_pull_6214-23discussion-5Fr50177334&d=BQMCaQ&c=54IZrppPQZKX9mLzcGdPfFD1hxrcB__aEkJFOKJFd00&r=h7heP8xwI1i_HikChvhFbEBurKirgfOCdwgBxB9lM8c&m=3mp2G8_Sxke75nhBolwdwstc3oMAUjC9tLx1TXs-zOQ&s=B16V8qiGgzSFnScE8BVq_qXfOLce5rPMxMqhWRthNbo&e=>
> :
>
> > @@ -406,7 +406,6 @@ class FEProblem :
> >
> >    // Function /////
> >    virtual void addFunction(std::string type, const std::string & name, InputParameters parameters, bool auto_parsed = false);
> > -  virtual bool hasFunction(const std::string & name, THREAD_ID tid = 0);
>
> This will break RELAP-7, we use this call.
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_idaholab_moose_pull_6214_files-23r50177334&d=BQMCaQ&c=54IZrppPQZKX9mLzcGdPfFD1hxrcB__aEkJFOKJFd00&r=h7heP8xwI1i_HikChvhFbEBurKirgfOCdwgBxB9lM8c&m=3mp2G8_Sxke75nhBolwdwstc3oMAUjC9tLx1TXs-zOQ&s=YpIEJpLMNcGqBtVVFqqpkSpRQFJMLYdhbz7Jx7swxM4&e=>
> .
>